llvm/lib/Target/SPIRV/SPIRVDuplicatesTracker.h

```diff
@@ -183,6 +184,9 @@ template <typename KeyTy> class SPIRVDuplicatesTrackerBase {
     }
     Storage[V][MF] = R;
     if (std::is_same<Function,
+                     typename std::remove_const<
+                         typename std::remove_pointer<KeyTy>::type>::type>() ||
+        std::is_same<Argument,
                      typename std::remove_const<
                          typename std::remove_pointer<KeyTy>::type>::type>())
```

Comment:
Use braces?

Reply:
I don't get it, what do you mean?

Reply:
I mean adding {} for this if body.

Reply:
It's a single-statement`if` w/o an `else', we should not use braces here.